components/MapView.js

```diff
@@ -14,9 +14,15 @@ import MapPolyline from './MapPolyline';
 import MapPolygon from './MapPolygon';
 import MapCircle from './MapCircle';
 import MapCallout from './MapCallout';
+import {
+  contextTypes,
+  airMapName,
+  AIRGoogleMapIsInstalled,
+  createNotSupportedComponent,
+} from './common';
 
 const viewConfig = {
-  uiViewClassName: 'AIRMap',
+  uiViewClassName: 'AIR?Map',
```

Comment:
Since we are using one component for two possible different bridged components, this would be `AIRMap` or `AIRGoogleMap`. I added the `?` to indicate to the user that we don't know which one it is.